.rubocop.yml

```diff
@@ -139,3 +139,19 @@ Style/MutableConstant:
 
 Style/FrozenStringLiteralComment:
   Enabled: false
+
+RSpec/LetSetup:
```

Comment:
Agree with all of these aside from this one - why wouldn't we ensure that our `let!`'s are referenced within the body of tests?

Reply:
What this does is complain if you use `let!`. I agree it shouldn't be overused, but I tend to think it's also a less verbose alternative to `before` and ivars.